src/rc_allocator.c

```diff
@@ -140,6 +139,16 @@ rc_allocator_get_capacity_virtual(allocator *a)
    return rc_allocator_get_capacity(al);
 }
 
+uint64
+rc_allocator_get_extent_num(rc_allocator *al, uint64 page_addr);
+
+uint64
+rc_allocator_get_extent_num_virtual(allocator *a, uint64 page_addr)
```

Comment:
Used on L2147 in `clockcache.c` for extra debugging info.

Reply:
I'm not a fan of this function. We don't use this outside of the allocator, and `extent_num` isn't a descriptive name. Rather than pipe all this through for a probably temporary `debug_assert`, I'd rather just have `base_addr / cache_extent_size(cfg)` in `clockcache.c`.

Reply:
Done. Withdrew `allocator_get_extent_num()`, `typedef uint64 (*get_num_fn)` and associated changes to pipe the extent_num thru a virtual fn handle.